scripts/wattlab_parity_watch_ghcr.py

```
@@ -0,0 +1,139 @@
+    return {"created": created, "tag": tag, "digest": digest, "top": rows[:5]}
+
+
+def main() -> int:
+    p = argparse.ArgumentParser(description=__doc__)
+    p.add_argument(
+        "--baseline-created",
+        required=True,
+        help="ISO created timestamp already tested; wake only if newer",
+    )
+    p.add_argument("--interval-sec", type=int, default=300)
+    p.add_argument("--state", type=Path, default=Path("/tmp/wattlab_parity_ghcr_watch.json"))
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/wattlab_parity_ghcr_watch.json"

(S108)